test/contracts/staking/staking-factory.test.ts

```diff
@@ -66,8 +66,13 @@ describe('Staking Pool Factory', () => {
             'Another erc20 Token',
             symbol
         )
-        stakingPoolFactory = await deployContract('StakingPoolFactory')
-        await stakingPoolFactory.initialize(treasury)
+
+        // eslint-disable-next-line no-console
+        console.log('deploying facotry')
```

Comment:
issue: no console logging please. If you. must, there's the logging already configured.

_There is a reason the linter checks for console out lines_ 😉